src/vm/docs/mod.rs

```diff
@@ -486,6 +486,17 @@ integer.",
     example: "(sha512 1) ;; Returns 0x6fcee9a7b7a7b821d241c03c82377928bc6882e7a08c78a4221199bfa220cdc55212273018ee613317c8293bb8d1ce08d1e017508e94e06ab85a734c99c7cc34",
 };
 
+const SHA512T256_API: SpecialAPI = SpecialAPI {
+    input_type: "buff|int",
+    output_type: "(buff 32)",
+    signature: "(sha512/256 value)",
+    description: "The `sha512/256` function computes `SHA512/256(x)` (the SHA512 algorithm with the 512/256 initialization vector, truncated
+to 256 bits) of the inputted value.
+If an integer (128 bit) is supplied the hash is computed over the little-endian representation of the
+integer.",
+    example: "(sha512/256 1) ;; Returns 0x6fcee9a7b7a7b821d241c03c82377928bc6882e7a08c78a4221199bfa220cdc55212273018ee613317c8293bb8d1ce08d1e017508e94e06ab85a734c99c7cc34",
```

Comment:
This isn't the right hash -- this value is 64 bytes.  The SHA512/256 of `\x01` is `2a8b71c345fac6e44dca4f7a762a560c3bdd7fa3001e84507c946a7d8fd46ee7`.

Reply:
Good catch -- yeah, the example code had been copied over from SHA512. Updated now.